config/app/com/microsoft/azure/iotsolutions/uiconfig/webservice/v1/controllers/SeedController.java

```diff
@@ -19,6 +20,7 @@ public SeedController(ISeed seed) {
         this.seed = seed;
     }
 
+    @Authorize("ReadAll")
```

Comment:
Is this a "ReadAll" permission? Does the webui call this method?

Reply:
the web ui doesn't call this method. We could remove the permission, although then a non-readonly or admin could make this request if they wanted to.